src/tests/unit/tests/background/telemetry/telemetry-client-provider.test.ts

```diff
@@ -38,7 +38,7 @@ describe('TelemetryClientProvider', () => {
             Mock.ofType<StorageAdapter>().object,
         );
 
-        expect(result).toBeInstanceOf(AppInsightsTelemetryClient);
+        expect(result).toBeInstanceOf(MultiplexingTelemetryClient);
```

Comment:
These test cases should probably be testing the number/types of the wrapped clients, rather than the type of the wrapping client, no?

Reply:
I get your point... although, that will either: break encapsulation or not really be a unit test, right?
I pushed a change with the first option (break encapsulation by accessing private member). Thoughts?

Reply:
I agree that this isn't great for encapsulation, but I also think what you're doing here is no better or worse than the original test, so approving the PR on that basis since everything else looks fine. One option might be to have the factory be responsible for outputting `TelemetryClient[]` rather than `TelemetryClient` and make it the caller's job to wrap it in the `MultiplexingTelemetryClient` ctor